servers/src/main/java/tachyon/master/TachyonMaster.java

```diff
@@ -121,12 +121,8 @@ public TachyonMaster(TachyonConf tachyonConf) {
 
     mIsStarted = false;
     mWebPort = webPort;
-    mMinWorkerThreads =
-        mTachyonConf.getInt(Constants.MASTER_MIN_WORKER_THREADS, Runtime.getRuntime()
-            .availableProcessors());
```

Comment:
This default is no longer the number of available processors. Is this intentional?